src/AppBundle/Twig/AppExtension.php

```diff
@@ -85,11 +86,20 @@ public function gitShortHash(): string
     }
 
     /**
-     * Get the full hash of the currently checkout-out Git commit.
+     * Get the full hash of the currently checked-out Git commit.
      * @return string
      */
     public function gitHash(): string
     {
         return exec('git rev-parse HEAD');
     }
+
+    /**
+     * Get the currently checked-out Git branch.
+     * @return string
+     */
+    public function gitBranch(): string
+    {
+        return exec('git rev-parse --symbolic-full-name --abbrev-ref HEAD');
```

Comment:
I think this should use [Process](https://symfony.com/doc/current/components/process.html) instead.

Reply:
Ah man, if we were on Symfony 4.2 we could use `Process::fromShellCommandline('...')->run()`. Instead we need to instantiate first, two liner versus one. Unless there's an easier way?

Reply:
3-liner even, but actually I think with Symfony 4.2 that would only bring it down to 2 lines (after `run()` you still need to call `->getOutput()`). No biggie